instances/service/CollaboratoryWiki.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/service/CollaboratoryWiki",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Service",
+  "definition": "EBRAINS Collaboratory Wiki",
```

Comment:
Maybe rather:
"The 'Collaboratory Wiki' is the main interface to access all other Collaboratory service provided by the EBRAINS research infrastructure."

Reply:
changed accordingly